src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/SymbolLoader.cs

```diff
@@ -94,44 +94,9 @@ public MiscSymFactory GetGlobalMiscSymFactory()
             return GlobalSymbolContext.GetGlobalMiscSymFactory();
         }
 
-        public AggregateType GetReqPredefType(PredefinedType pt)
-        {
-            AggregateSymbol agg = GetTypeManager().GetReqPredefAgg(pt);
-            if (agg == null)
-            {
-                Debug.Assert(false, "Required predef type missing");
-                return null;
-            }
-            AggregateType ats = agg.getThisType();
-            return ats;
-        }
+        public AggregateSymbol GetPredefAgg(PredefinedType pt) => GetTypeManager().GetPredefAgg(pt);
 
-        public AggregateSymbol GetOptPredefAgg(PredefinedType pt)
-        {
-            return GetTypeManager().GetOptPredefAgg(pt);
-        }
-
-        public AggregateType GetOptPredefType(PredefinedType pt)
-        {
-            AggregateSymbol agg = GetTypeManager().GetOptPredefAgg(pt);
-            if (agg == null)
-                return null;
-            AggregateType ats = agg.getThisType();
-            return ats;
-        }
-
-        public AggregateType GetOptPredefTypeErr(PredefinedType pt)
-        {
-            AggregateSymbol agg = GetTypeManager().GetOptPredefAgg(pt);
-            if (agg == null)
-            {
-                getPredefTypes().ReportMissingPredefTypeError(ErrorContext, pt);
-                return null;
-            }
-
-            AggregateType ats = agg.getThisType();
-            return ats;
-        }
+        public AggregateType GetPredefType(PredefinedType pt) => GetPredefAgg(pt).getThisType();
```

Comment:
Same for similar functions, since we're refactoring them.

---
In reply to: [129394862](https://github.com/dotnet/corefx/pull/22471#discussion_r129394862) [](ancestors = 129394862)